ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -39,8 +39,9 @@ to answer all of them.
 
 ### Additional Reviews
 
-5. Does this document need review from other IETF working groups or external
-   organizations? Have those reviews occurred?
+5. Does this document closely relate to work in other IETF working groups or
```

Comment:
This is now very different from the original

``` 
(5) Do portions of the document need review from a particular or
from broader perspective, e.g., security, operational complexity,
AAA, DNS, DHCP, XML, or internationalization? If so, describe the
review that took place.
``` 

The original is closer to "Have you consulted your YANG doctor", while the new text is not addressing foundational technologies at all but instead asks about related (presumably ongoing) work.  What kind of relation?
Addressing a related problem, but in a different way, based on different foundational technologies?